fly/commands/hijack.go

```diff
@@ -30,6 +31,7 @@ type HijackCommand struct {
 	StepName       string                   `short:"s" long:"step"                              description:"Name of step to hijack (e.g. build, unit, resource name)"`
 	StepType       string                   `          long:"step-type"                         description:"Type of step to hijack (e.g. get, put, task)"`
 	Attempt        string                   `short:"a" long:"attempt" value-name:"N[,N,...]"    description:"Attempt number of step to hijack."`
+	TimeOut        string                   `short:"o" long:"timeout"                           description:"Time to wait for container."`
```

Comment:
```suggestion
	Timeout        time.Duration                   `short:"o" long:"timeout"                           description:"Time to wait for container."`
```
Since the flag represents a "timeout" and not a "time out", and you can leverage the go-flags library to do the time parsing on your behalf!

Reply:
Thanks for the suggestion, didn't realize this was possible. I've updated the pull request to reflect this change.